### PR TITLE
fix: add referral link to sendgrid users

### PR DESCRIPTION
**Pull request by rebelchris**
Ensure new parsed people get their referral link added to the new custom field in Sendgrid.
Next step is to do a batch CSV to insert all existing users with this field set.

WT-1978 #done

### CODE REVIEW

File: src/common/mailing.ts [hunk 1]
```diff
@@ -90,10 +91,18 @@ interface EmailContact {
...
+    userId: profile.id,
+  });
+
   const contact: EmailContact = {
     id: contactId,
     email: profile.email,
-    custom_fields: { e1_T: profile.id },
+    custom_fields: {
+      e1_T: profile.id,
+      e2_T: genericInviteURL.toString(),
```
Comment (idoshamun) [inline src/common/mailing.ts:104]: Just to confirm, have you created it on sendgrid already? (the field I mean)

**Review by idoshamun — APPROVED** [review]
Looks great! Just check my comment

File: src/common/mailing.ts (hunk 1 above)
Comment (rebelchris) [reply src/common/mailing.ts:104]: Yep and run the initial CSV already

Comment (idoshamun) [reply src/common/mailing.ts:104]: Perfect!

**Pull request merged**